markov.py

```diff
@@ -132,16 +133,31 @@ def help(message):
         "{remove_command}: {username} will remove messages from chat.\n"
         "{version_command}: {username} will state its current version.\n"
         "{flush_command}: {username} will clear its cache.\n"
+        "{start_command}: {username} will display quickstart info.\n"
         "{help_command}: {username} will print this help message!"
     )
     output_text = help_text.format(
         username=username, sentence_command=sentence_command,
         remove_command=remove_command, version_command=version_command,
-        flush_command=flush_command, help_command=help_command
+        flush_command=flush_command, start_command=start_command,
+        help_command=help_command
     )
     bot.reply_to(message, output_text)
 
 
+@bot.message_handler(commands=[settings.START_COMMAND])
+def start(message):
+    bot.reply_to(message, f"Welcome to MarkovBot, a Telegram bot that writes "
+                          f"like you do by using Markov chains. To get "
```

Comment:
looks great! just a few changes: we dont need to enable the bot anymore. just add the bot to the group and send some messages. you can also remove the part about disabling  privacy.

Reply:
Gotcha, give me one moment to update that.